packages/cli/src/handlers/preview.ts

```diff
@@ -314,7 +315,7 @@ export const startPreviewHandler = async (
         await deploy(explores, {
             ...options,
             projectUuid: previewProject.projectUuid,
-            ignoreErrors: true,
+            ignoreErrors: !options.raiseErrors,
```

Comment:
We need to remove `ignoreErrors: !options.raiseErrors,` and `ignoreErrors: true` from all calls to `await deploy(...)`.